writing-visualising/dashboards.qmd

```diff
@@ -28,7 +28,9 @@ These standards are constantly evolving, and all feedback and contributions are
 
 All dashboards should have a clear use case and reasoning for why they are required. They also need to meet the [latest accessibility regulations](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag) for public sector services.
 
-Dashboards, along with the Official Statistics that they supplement, are a digital service that we provide, and we should ensure we follow the [Service Standard](https://www.gov.uk/service-manual/service-standard). The standard covers a number of aspects that all producers should be considering when developing a dashboard, and strongly influence the guidance on this page.
+Dashboards, along with the Official Statistics that they supplement, are a digital service that we provide, and we should ensure we follow the [Service Standard](https://www.gov.uk/service-manual/service-standard), and the [Office for Statistics Regulation (OSR) regulatory guidance on dashboards](https://osr.statisticsauthority.gov.uk/publication/regulatory-guidance-dashboards/).
```

Comment:
Is it worth adding a note that teams shouldn't add the "Official stats" badging directly to their dashboards, given that they're secondary tools alongside official stats rather than the official stats themselves. And directly referencing the guidance that FE found and highlighting how this approach is actually consistent with that guidance?